.buildkite/release-pipelines/code-freeze.yml

```diff
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+---
+
+# This pipeline is meant to be triggered by the MC scenario in ReleasesV2
+
+steps:
+  - label: "🚂 Code Freeze"
+    command: |
+      echo '--- 🤖 Use bot for Git operations'
+      source use-bot-for-git
+
+      echo '--- :ruby: Setup Ruby Tools'
+      install_gems
+
+      echo '--- 🔐 Access Secrets'
+      bundle exec fastlane run configure_apply
```

Comment:
I don't think this is something we've ever done in the release pipelines, but how do you feel about extracting these common steps in a script to `source`?

Could be a followup / mini-project to log and pick up once someone has time to implement consistently. Just thought I'd toss the idea there for DRYing.

Reply:
Yeah, as I copy/pasted the `release-pipelines/code-freeze.yml` multiple times to create the other `release-pipelines/` and seeing how similar they all were I thought about it too; a good idea to implement cross-repo in a subsequent iteration indeed!